### PR TITLE
Adding imu calibrations scripts (do_calib and apply_calib)

### PR DESCRIPTION
**Pull request by bjkim00**
Adding bash scripts to make running imu calibrations less manual

### CODE REVIEW

File: do_calib.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	echo ""
+}
+
+array=()
+read -p "How many IMUs would you like to calibrate: " imu_num
+echo "Number of IMUs: $imu_num"
+
+for i in $(seq 1 $imu_num); do
+	#echo "HI"
+	read -p "Enter the imu number: " num
```
Comment (watakandai) [inline do_calib.sh]: does this mean we have to input the imu number for n imu times?

Comment (bjkim00) [reply do_calib.sh]: Currently yes, do you want me to change the way it's being done?

**Comment by peasant98** [comment]
Could you add a quick "how to run" in the docs somewhere?

File: do_calib.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+array=()
+
+defnum=0
+defimu=0
+
+num_imus=${1:-$defnum}
+i_imu=${2:-$defimu}
+
+if [ $num_imus -gt $defnum ]
```
Comment (watakandai) [inline do_calib.sh:28]: looks good. Does it work? and if so, can you add some comments?

File: do_calib.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	echo ""
+}
+
+array=()
+
+defnum=0
+defimu=0
+
+num_imus=${1:-$defnum}
+i_imu=${2:-$defimu}
```
Comment (watakandai) [inline do_calib.sh:23]: what does this line do

**Review by watakandai — CHANGES_REQUESTED** [review]
Also please add dependency to install script. 
I think you just need to git clone the repo under catkin_ws/src and run catkin build.

**Comment by watakandai** [comment]
Please fix the flake8 and it's good to be merged :+1:

**Review by watakandai — APPROVED** [review]

**Pull request merged**